src/Microsoft.Azure.SignalR.Protocols/ConnectionMessage.cs

```diff
@@ -136,5 +136,10 @@ public ConnectionDataMessage(string connectionId, ReadOnlySequence<byte> payload
         /// Gets or sets the binary payload.
         /// </summary>
         public ReadOnlySequence<byte> Payload { get; set; }
+
+        /// <summary>
+        /// Gets or sets the sequence id.
+        /// </summary>
+        public int SeqId { get; set; } = 0;
```

Comment:
long? Or we're ready to deal with wrapping around max int?

Reply:
I don't think a client connection would hold over 0.2b messages in its lifecycle?

Reply:
It should be a long

Reply:
Also the property should be SequenceId

Reply:
2bln messages is just ~20 days with ~1K messages/second. Might be not that common but not unrealistic at all.